README.md

```diff
@@ -64,38 +67,44 @@ var handle = client.execute({
 handle.stream(function(err, data) { console.log(data); });
 ```
 
-Object `options` is an optional map and may contains following fields:
+The `options` object is an optional map and can contain following fields:
 
-- **signalflowEndpoint** - string, `wss://stream.us0.signalfx.com` by default. Override this if you are in a different realm than `us0`.
-- **apiEndpoint** - string, `https://api.us0.signalfx.com` by default. Override this if you are in a different realm than `us0`.
-- **webSocketErrorCallback** - function, Throws an Error event by default. Override this if you want to handle a websocket error differently.
+| Field                    | Type     | Default                                                                                            |
+|--------------------------|----------|----------------------------------------------------------------------------------------------------|
+| `signalflowEndpoint`     | string   | `wss://stream.us0.signalfx.com`. If you aren't in `us0`, change the realm.                         |
+| `apiEndpoint`            | string   | `https://api.us0.signalfx.com`. If you aren't in `us0`, change the realm.                          |
+| `webSocketErrorCallback` | function | Throws an error event. If you want to handle the websocket error differently, change the behavior. |
```

Comment:
Would it make sense to add a description for each fields?

Reply:
Sure, perhaps the descriptions from https://dev.splunk.com/observability/docs/apibasics/api_list/#Table-of-endpoints?